setup.py

```diff
@@ -20,6 +21,21 @@ def run_tests(self):
         sys.exit(errno)
 
 
+install_requires=['PyYAML', 'wrapt', 'six>=1.5']
+
+
+extras_require = {
+    ':python_version in "2.4, 2.5, 2.6"':
+        ['contextlib2', 'backport_collections', 'mock'],
+    ':python_version in "2.7, 3.1, 3.2"': ['contextlib2', 'mock'],
+}
+
+
+if 'bdist_wheel' not in sys.argv:
```

Comment:
can we not do a version check here too. Won't this make it do it unconditionally?